docs/getting_started/binaries.rst

```diff
@@ -88,16 +88,36 @@ The Linux binary builds can be run on Windows using the
 when solvers depend on POSIX standards.  Currently the optional PETSc solver is
 the only one which does not run on Windows.
 
-In the WSL environment, download and extract the desired Linux release containing
-the solver you would like to run.  In the Windows IDAES binary directory, create a
-batch file with the format ``{solver}_wsl.bat`` for example for PETSc
-``petsc_wsl.bat``.
+In the WSL environment, download and extract the desired Linux release of
+the solver (i.e. PETSc) you would like to run following the instructions under manual
+installation above. Install required libraries libfortran, libblas, liblapack i.e.
+``apt-get install liblapack3 libblas3 ligbomp1 libgfortran5``.
+
+In the WSL environment you can check that PETSc is installed correctly. In your 
+desired Linux release navigate to the folder containint the compiled PETSc binary
+and run the command ``./petsc --version``. If setup correctly details of the PETSc
+release will be printed on the screen.
 
-Assuming you have put the binary file in the ``$HOME/loacl/bin`` and your WSL
-user name is ``john``, and the distribution installed is ``Ubuntu-20.04`` the
-contents of the batch file to run the petsc are below:
+In the Windows IDAES binary directory, create a
+batch file with the format ``{solver}_wsl.bat`` for example for PETSc
```

Comment:
The command ``idaes bin-directory`` tells you where the bin directory is.

Reply:
Updated the documentation to capture this.